src/domains/job/recruitment/components/form/WriteRecruitmentPostingForm.jsx

```
@@ -86,6 +86,7 @@ const WriteRecruitmentPostingForm = ({ onSubmit, defaultValues }) => {
       <div className='my-[30px]'>
         <PostContent control={control} errors={errors} />
       </div>
+      <button type='button' onClick={onSave} hidden />
```

ESLint

[error] 'onSave' is not defined.

(no-undef)